README.md

```diff
@@ -148,6 +148,19 @@ Use this command in your `.zshrc` to load Spaceship as prompt theme:
 zplug denysdovhan/spaceship-prompt, use:spaceship.zsh, from:github, as:theme
 ```
 
+### Linux package manager
+
+#### Arch Linux
+
+Install the latest master from the [AUR package
+`spaceship-prompt-git`](https://aur.archlinux.org/packages/spaceship-prompt-git/):
```

Comment:
Why these are on separate lines ? Would be better if they're on the same line and linked only to `spaceship-prompt-git`,

```diff
-Install the latest master from the [AUR package
-`spaceship-prompt-git`](https://aur.archlinux.org/packages/spaceship-prompt-git/):
+Install the latest master from the AUR package [`spaceship-prompt-git`](https://aur.archlinux.org/packages/spaceship-prompt-git/):
```

Reply:
Sorry about that, my editor wraps at 80 characters and I didn't notice that there are long lines elsewhere. I'm fine with that linking change too.